persistence-dse-6.8/src/main/java/io/stargate/db/dse/DsePersistenceActivator.java

```diff
@@ -38,7 +39,8 @@
   private File baseDir;
   private volatile BundleContext context;
 
-  private Config makeConfig() throws IOException {
+  @VisibleForTesting
+  public static Config makeConfig(File baseDir) throws IOException {
     Config c = new Config();
 
     // Throw away data directory since stargate is ephemeral anyway
```

Comment:
I believe you forgot to remove the assignment of `baseDir` here since it's now an argument.

Reply:
thx. will fix

Reply:
fixed